localbuild/meta.yaml

```diff
@@ -39,16 +39,16 @@ requirements:
     - chameleon
     - execnet
     - fastkml =0.11
-    - shapely <2.0.0
+    - shapely >=2.0.0
     - pygeoif <1.0.0
     - isodate
     - lxml
     - netcdf4
     - hdf4
     - pillow
     - pytz
-    - pyqt >=5, <5.13
-    - qt >=5.10, <5.13
+    - pyqt >=5.15.0
```

Comment:
this is needed to get recent python versions used. This also enables updating a lot dependent packages.